src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -1062,3 +1066,63 @@ def wrapper(*args, **kwargs):
         return result
 
     return wrapper
+
+
+def execute_flow(
+    flow_file: Path,
+    working_dir: Path,
+    output_dir: Path,
+    connections: dict,
+    inputs: Mapping[str, Any],
+    *,
+    func: Optional[str] = None,
+    storage: Optional[AbstractRunStorage] = None,
+    raise_ex: bool = True,
```

Comment:
raise_ex might be a legacy parameter, is there anywhere to use True?

Reply:
Shouldn't we set this parameter to true for the executor server?

Reply:
Updated.